profiler/controllers.py

```diff
@@ -1,70 +1,175 @@
-import os
+# coding: utf-8
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+# Copyright 2014 Anybox <http://anybox.fr>
+# Copyright 2016 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+import errno
```

Comment:
Follow the [module template](https://github.com/OCA/maintainer-tools/tree/master/template)
I means, this file should be: `controllers/profiler.py`